spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -955,10 +961,10 @@ def highlight_canvas(self, highlight):
             # See spyder-ide/spyder#10255.
             if is_dark_interface():
                 self.canvas.setStyleSheet(
-                        "FigureCanvas{border: 2px solid %s;}" % "#148CD2")
+                    "FigureCanvas{border: 2px solid %s;}" % "#148CD2")
```

Comment:
Where does this value come from? Maybe adding a reference to QDarkStyle could help, so that eventually we can remove this and use a centralized color handler.